src/ldc/attributes.d

```diff
@@ -223,3 +223,23 @@ immutable weak = _weak();
 private struct _weak
 {
 }
+
+///Readability aliases for compute
```

Comment:
like `@compute(hostAndDevice) module foo;`

>probably better to make them members of the struct.

I'm not quite sure how you mean?

like 
```D
struct compute {
  int alsoProduceHostCode;
  enum hostAndDevice = 1;
  enum deviceOnly = 0;
}
```
?
Then you'd have to go like 
`@compute(compute.deviveOnly) module foo;`
wouldn't you?
Anyway I'm open to suggestion for improving encapsulation/readability.